Monika After Story/game/script-topics.rpy

```diff
@@ -5587,3 +5587,63 @@ label monika_dreaming:
     m 1a "But can you just imagine what it would be like to meet me in your dreams [player], any night that you wanted to?"
     m 1j "I hope that someday we can make your dreams about us a reality, my love."
     return
+
+init 5 python:
+     addEvent(Event(persistent.event_database, eventlabel="monika_star_gazing", category=["romance"], prompt="Star Gazing", random=True))
+
+label monika_star_gazing:
+    m 1a "Hey [player], have you ever gone Star Gazing?"
+    m 3a "Like, taking some time out of your evening to look at the stars in the sky."
+    m 1b "It's a rather nice feeling and surprisingly relaxing, to stare at the beauty of stars."
+    m 3b "It's also a good way to relieve stress and clear your head."
+    m 3b "With your mind focused on the sky and becoming distracted from unhappy thoughs and feelings."
```

Comment:
Expressions don't need to be repeated if used prior. You can omit the expression if you intend to use the same one.
`m "With you mind`
Like that. I'll not comment on the others but it's a repeated issue in all of the topics.

Also I believe you intended to write thoughts and not thoughs.

Reply:
Not sure what you mean by this, if it's an issue affecting all my topics i'd like to fix it.